# match a PeoplePoseList found by some detection
# against a set of reference (known) PeoplePoseList

# the detected PeoplePoseList.
# the method is accessible via new_ppl.method, ex: "face_recognition_eigen"
people_msgs/People new_ppl
# the reference (known) PeoplePoseList
people_msgs/People tracks

---

# true if the matching between both PPL was a success.
# if true, "costs" is supposed to be of good size (see below)
# and filled with meaningful values.
# otherwise, "costs" will be discarded (wont even be accessed).
bool match_success

# "costs" should be of size (|new_ppl| x ntracks),
# ie new_ppl.poses.size() x tracks.people.size(),
# the costs are obtained by costs[detected_pp_idx][track_idx]
float64[] costs

# the attributes to add to new_ppl, that were computed by the node
# to be valid, the vector should be of size |new_ppl.poses|
string[] new_ppl_added_tagnames
string[] new_ppl_added_tags
uint32[]    new_ppl_added_indices

# the attributes to add to tracks,  that were computed by the node
# to be valid, the vector should be of size |tracks.people|
string[] tracks_added_tagnames
string[] tracks_added_tags
uint32[]    tracks_added_indices
